CMakeLists.txt

```diff
@@ -99,13 +104,15 @@ set(CMAKE_MODULE_LINKER_FLAGS "${CMAKE_MODULE_LINKER_FLAGS} -Wl,-z,nodelete")
 # This is the compiler pass that we later load into clang or opt. If LLVM is
 # built without RTTI we have to disable it for our library too, otherwise we'll
 # get linker errors.
-add_library(Symbolize MODULE
+add_library(Symcc MODULE
   compiler/Symbolizer.cpp
   compiler/Pass.cpp
   compiler/Runtime.cpp
   compiler/Main.cpp)
+
+set_target_properties(Symcc PROPERTIES OUTPUT_NAME "symcc")
```

Comment:
I wonder if we should keep the library name `libSymbolize.so` :thinking: These are breaking changes anyway, but maybe we can save the users some work by not changing the library name. (My employer is one such user, and I'm the person who will have to fix the build scripts, so I'm not being entirely altruistic here :upside_down_face:)

Reply:
I knew when writing this line it would be problematic :)
I was thinking it would be nice to keep consistent with other libraries in general, which seems to be `lib<lowercase_project_name>.so`.
If it is too much work to change this, let's keep the original name.

Reply:
Is this really a big change for the "users"? Taking the opportunity to changing to something that makes more sense seem appealing ;)